events/hfe_events_pirate.txt

```diff
@@ -502,7 +503,7 @@ country_event = {
 	trigger = {
 		num_pops > 0
 		or = {
-			num_pops > 10
+			num_pops > 1
```

Comment:
10 is better value

Reply:
No way. Pirates never advertise their slaves enough and they rarely abduct.

Reply:
Pirates without pops abduct even worse

Reply:
Ten Pops is crazy. They almost never have ten Pops in storage. It's been a flaw from day one. I set it so that you can actually get notified of a chance to buy slaves from pirates.